docs/Reference/Console/GetStarted/addDevice.md

```diff
@@ -0,0 +1,49 @@
+--8<-- "References/abbreviations.md"
+
+!!! warning "Warning - Console in Alpha Development"
+    Console is currently under development. The current available tags for download are Alpha version code. This could mean that certain features may not function yet, visual look and feel may change, or bugs/errors may occur. It is not recommended for use in Production deployments. Follow along our [Feature Backlog for future releases and feature updates](https://github.com/orgs/open-amt-cloud-toolkit/projects/10)
+
+1. Open the Console UI in the browser.
```

Comment:
When you launch the console application the UI should automatically open in the default browser

Reply:
That's mentioned in the prior page.  I was basing this one on users navigating away from Console to get RPC and activate device, then returning back to Console.  BUT that page doesn't exist for now... I reworded this.